graphql/user/fields.ts

```diff
@@ -136,13 +137,27 @@ export class UserFieldsResolver {
     @Authorized(Role.OWNER, Role.ADMIN)
     @LimitedQuery()
     async concreteNotifications(
```

Comment:
Maybe there are other scenarios in which one wants alls sent notifications (e.g. for a GDPR data export or to show notifications sent via email etc.). So maybe it makes sense to filter through a parameter. But YAGNI ...